src/t_stream.c

```diff
@@ -2128,17 +2128,20 @@ void xreadCommand(client *c) {
              * of the stream and the data we extracted from it. */
             if (c->resp == 2) addReplyArrayLen(c,2);
             addReplyBulk(c,c->argv[streams_arg+i]);
-            int created = 0;
             streamConsumer *consumer = NULL;
-            if (groups) consumer = streamLookupConsumer(groups[i],
-                                                        consumername->ptr,
-                                                        SLC_NONE,
-                                                        &created);
             streamPropInfo spi = {c->argv[i+streams_arg],groupname};
-            if (created && noack)
-                streamPropagateConsumerCreation(c,spi.keyname,
-                                                spi.groupname,
-                                                consumer->name);
+            if (groups) {
+                consumer = streamLookupConsumer(groups[i],
+                                     consumername->ptr,1);
```

Comment:
this one is even shorter than 80.
```suggestion
                consumer = streamLookupConsumer(groups[i],consumername->ptr,1);
```

Reply:
done